.github/workflows/zos-build.yml

```diff
@@ -40,7 +40,13 @@ jobs:
         run: npm ci --workspaces=false
 
       - name: Build on z/OS
-        run: npm run z:all
+        uses: nick-fields/retry@v3
+        with:
+          command: npm run z:all
+          max_attempts: 3
+          new_command_on_retry: npm run z:build && npm run z:artifacts
+          retry_on_exit_code: 11 # Ignore transient SIGSEGV error on Go build
```

Comment:
Hey, why are we ignoring these? 🤔

Reply:
Related to #229 - they are transient errors on the Marist system that frequently occur when the build should succeed. Since we only retry on this specific exit code, any other Go build errors should still cause CI to fail.